README.md

```diff
@@ -1,5 +1,31 @@
 # SMLHelper
-A complete library that helps you out on your Subnautica modding adventures by making adding new items, models, sprites a LOT easier. Check out the wiki page.
+SMLHelper is a complete library that helps you out on your Subnautica modding adventures by making adding new items, changing items, adding models, sprites a LOT easier. Check out the wiki page for details on how to use it.
 
-# Contributing
+## Contributing
 We would love to have people contribute to SMLHelper. To get started, first fork the repo and then clone it, and then copy Assembly-CSharp.dll and Assembly-CSharp-firstpass.dll (located in your Subnautica_Data/Managed folder) to the Dependencies folder. Then, load up the solution, make your edits and PR them!
+
+## Alphabetical list of mods requiring SMLHelper
+_(Last updated: July 2nd 2018)_
+
+* [Base Clocks](https://www.nexusmods.com/subnautica/mods/75) by Fenolpthalien (mod)
+* [Base Light Switch](https://www.nexusmods.com/subnautica/mods/46) by ahk1221 (mod)
+* [Custom Item Sizes](https://www.nexusmods.com/subnautica/mods/79) by AlexejheroYTB (mod)
+* [CustomFood](https://www.nexusmods.com/subnautica/mods/73) by yenzgaming (mod)
+* [CustomHullPlates](https://www.nexusmods.com/subnautica/mods/95) by ahk1221 (mod)
+* [Deconstruct Fire Extinguishers](https://www.nexusmods.com/subnautica/mods/103) by AlexejheroYTB (mod)
+* [Decorations Mod](https://www.nexusmods.com/subnautica/mods/102) by OSubMarin (mod)
+* [Egg Info - Improved Alien Containment](https://www.nexusmods.com/subnautica/mods/81) by yenzgaming (mod)
+* [ExteriorPlantPots](https://www.nexusmods.com/subnautica/mods/64) by ahk1221 (mod)
+* [More Cyclops Upgrades](https://www.nexusmods.com/subnautica/mods/101) by PrimeSonic (mod)
+* [More Ingots](https://www.nexusmods.com/subnautica/mods/60) by AlexejheroYTB (mod)
+* ~~[More Hull Plates](https://www.nexusmods.com/subnautica/mods/82) by AlexejheroYTB (mod)~~
+* [MoreSeamothUpgrades](https://www.nexusmods.com/subnautica/mods/98) by ahk1221 (mod)
+* [Refillable Oxygen Tanks](https://www.nexusmods.com/subnautica/mods/67) by newman55 (mod)
+* [Replenish Reactor Rods](https://www.nexusmods.com/subnautica/mods/106) by AlexejheroYTB (mod)
+* [Stargate themed hull plates](https://www.nexusmods.com/subnautica/mods/96) by IngloriousBastard2 ([CustomHullPlates](https://www.nexusmods.com/subnautica/mods/95) hull plate pack)
+* [StOrM'S_World](https://www.nexusmods.com/subnautica/mods/54) by StOrM0HyDrA0DBS (saved game)
```

Comment:
I'm not sure whether to include stargate themed hull plates and storm's world in this list. for now I've added (saved game) and (customhullplates hull plate pack) at the end of those lines + I've added (mod) at the end of every other line. if you think this doesn't look good, let me know and i'll change it.